.github/workflows/unit-test.yml

```diff
@@ -62,8 +62,9 @@ jobs:
           kubectl: 1.18.4
           kubeval: v0.16.1
           command: |
-            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/*.yml
-            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/kustomize/base/cloudbeat*.yml
+            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/*.yml --ignore-missing-schemas
+            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/eks/*.yml --ignore-missing-schemas
+            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/kustomize/base/*.yml --ignore-missing-schemas
 
```

Comment:
I would add the overlays too, wdyt?
```
kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/kustomize/overlays/cloudbeat-eks/*.yml --ignore-missing-schemas
kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/kustomize/overlays/cloudbeat-vanilla/*.yml --ignore-missing-schemas
kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/kustomize/overlays/test/*.yml --ignore-missing-schemas
```

Reply:
they will result in only warnings in our case, but will still prevent any breaking change (errors) no?

Reply:
not sure, I tried to remove essential fields for the deployment and still I get only warnings, it takes a really nasty change to make it notifies of an error.

In general, I think that since they are partial files, they should not be tested, you will always get a warning on these.
But I guess it is a matter of perspective, if you still think it is better doing it, I can do it.